### DISCUSSION TITLE
How to reveal the whatsapp group ids?

### DISCUSSION THREAD

**Discussion by HorstBaumann** [42wim/matterbridge#1534] [Help / Q&A]
since updating to 1.22.3, matterbridge does not reveal the ids of the grupos/channels my whatsapp account is a member of

when I start mattebridge with matterbridge-1.22.3-windows-64bit.exe -conf matterbridge.toml
the "INFO whatsapp"-part is missing

1.22.1 showed it, but isn't no longer working

matterbridge (1.22.3) works fine, but I need to get the id of a new group, I just joined

**Comment by 42wim** [comment]
It only shows it when you specify an incorrect config.

**Reply by ndurchx** [reply]
Now I have the problem, that matterbridge is not showing the channel ids anymore. Instead it shows the channel name a second time:

level=fatal msg="Starting gateway failed: Bridge xxx failed to join channel: group name might change. Please configure gateway with channel="CHANNELNAME" instead of channel="CHANNELNAME"" prefix=main

**Reply by beedell-roke** [reply]
@42wim, sounds like a useful FR, then.

**Comment by gamersindo1223** [comment]
@HorstBaumann send me ur config
